Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -47,6 +47,7 @@
 --shortoptionals always # typeSugar
 --semicolons never # semicolons
 --doccomments preserve # docComments
+--urlmacro #URL,URLFoundation
```

Comment:
Since `URLFoundation` is only available internally, we shouldn't actually enable this option in the `airbnb.swiftformat` here in this repo

```suggestion
```